pages/integrations/sso/google_workspace_saml.md.erb

```diff
@@ -8,6 +8,8 @@ To complete this tutorial, you will need admin privileges for both Google Worksp
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+Once this process is complete, users can sign into Buildkite by clicking the icon in the "9 dots" applications menu of the top bar available on many Google products.
```

Comment:
I considered putting a screenshot here, but... the one I took on linux looks a bit ugly:

![google-initiated-login](https://user-images.githubusercontent.com/8132/140050492-a0459638-5435-4108-bfe6-34264e595306.png)

I don't think it's critical. The info added in this PR is still an improvement, even without the image.

Reply:
Got it! So will you be OK with me going over your PR + possibly adding a prettier screenshot & merging, @yob? No time pressure, just want to figure out what to do with this piece of docs whenever you're ready.